src/rust/ide/view/graph-editor/src/lib.rs

```diff
@@ -380,6 +380,10 @@ ensogl::define_endpoints! {
         /// Switches the selected visualisation to/from fullscreen mode.
         toggle_fullscreen_for_selected_visualization(),
 
+        // === Scene Navigation ===
+
+        enable_navigation(),
+        disable_navigation(),
```

Comment:
What this is about? Seems very strange. If this is a hack, please describe in comment why it is needed, what happens when its not present, and link to an issue about it.

Reply:
I've added docs. Essentially, this is about locking the navigation of the scene, disabling panning/zooming etc.